src/components/UserTabs/UserCard/UserCard.module.css

```diff
@@ -59,11 +78,96 @@
   color: var(--color-black);
 }
 
-.user_recepies {  
+.user_recepies {
   margin-bottom: 8px;
-    font-weight: 500;
+  font-weight: 500;
   font-size: 12px;
   line-height: 1.5;
   letter-spacing: -0.02em;
   color: var(--color-main-60);
 }
+
+
+@media screen and (min-width: 768px) {
+
+
+  .follow_list_item:not(:last-child)::after {
+    bottom: 0;
+  }
+
+  .follow_list_item:not(:last-child) {
+    padding-bottom: 40px;
+  }
+
+  .recipes_pictures_list {
+    display: flex;
+    width: 324px;
+    gap: 12px;
+  }
+
+  .recipes_pictures_item {
+    width: calc((100% - 24px) /3);
+    height: 100px;
+    border-radius: 15px;
+    overflow: hidden;
+  }
+
+  .user_avatar {
+    width: 85px;
+    height: 85px;
+  }
+
+  .user_link_arrow {
+    width: 42px;
+    height: 42px;
+  }
+
+  .user_name {
+    font-size: 20px;
+    line-height: 1.2;
+  }
+
+  .user_recepies {
+    font-size: 14px;
+    line-height: 1.43;
+  }
+
+  .selected_user {
+    width: 16px;
+    height: 16px;
+  }
+
+}
+
+@media screen and (min-width: 768px) and (max-width: 1439px) {
+  .recipes_pictures_item:nth-child(n + 4) {
+    display: none;
```

Comment:
would be better not to render pictures on this screen size by using useTabletMediaQuery in your component